appliances/MinIO/appliance.sh

```diff
@@ -0,0 +1,331 @@
+# ---------------------------------------------------------------------------- #
+# Copyright 2024, OpenNebula Project, OpenNebula Systems                  #
+#                                                                              #
+# Licensed under the Apache License, Version 2.0 (the "License"); you may      #
+# not use this file except in compliance with the License. You may obtain      #
+# a copy of the License at                                                     #
+#                                                                              #
+# http://www.apache.org/licenses/LICENSE-2.0                                   #
+#                                                                              #
+# Unless required by applicable law or agreed to in writing, software          #
+# distributed under the License is distributed on an "AS IS" BASIS,            #
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.     #
+# See the License for the specific language governing permissions and          #
+# limitations under the License.                                               #
+# ---------------------------------------------------------------------------- #
+
+### Important notes ##################################################
+#
+# The contextualization variable 'ONEAPP_SITE_HOSTNAME' IS (!) mandatory and
+# must be correct (resolveable, reachable) otherwise the web will be broken.
+# It defaults to first non-loopback address it finds - if no address is found
+# then the 'localhost' is used - and then wordpress will function correctly
+# only from within the instance.
+#
+# 'ONEAPP_SITE_HOSTNAME' can be changed in the wordpress settings but it should
+# be set to something sensible from the beginning so you can be able to login
+# to the wordpress and change the settings...
+#
+### Important notes ##################################################
+
```

Comment:
Can't find `set -e` or `set -o errexit` and `set -o pipefail -o nounset` I think you should consider using them for safety (somebody that will someday debug this code will appreciate it :hugs: ). :point_up::relieved: